Maestro/subscriptions.json

```diff
@@ -365,6 +365,36 @@
         }
       }
     },
+    // Update dependencies in CoreCLR release/2.2
```

Comment:
It feels like we need to generalize these triggers a little more so we don't have to duplicate all the information each time we branch. Something to keep in mind with Maestro++ and dependency updating.